src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -557,6 +557,23 @@ describe('manage nsfs cli account flow', () => {
             expect(JSON.parse(res.stdout).error.message).toBe(ManageCLIError.InvalidArgumentType.message);
         });
 
+        it('cli unset new_buckets_path', async () => {
+            const { name } = defaults;
+            const account_options = { config_root, name, new_buckets_path: '\'\''};
+            const action = ACTIONS.UPDATE;
+            await exec_manage_cli(type, action, account_options);
+            let new_account_details = await read_config_file(config_root, CONFIG_SUBDIRS.ACCOUNTS, name);
+            expect(new_account_details.nsfs_account_config.new_buckets_path).toBeUndefined();
+            expect(new_account_details.allow_bucket_creation).toBe(false);
+
+            //set new_buckets_path value again
+            account_options.new_buckets_path = defaults.new_buckets_path;
+            await exec_manage_cli(type, action, account_options);
+            new_account_details = await read_config_file(config_root, CONFIG_SUBDIRS.ACCOUNTS, name);
+            expect(new_account_details.nsfs_account_config.new_buckets_path).toBe(defaults.new_buckets_path);
+            expect(new_account_details.allow_bucket_creation).toBe(true);
```

Comment:
I'm not sure you need this inside the same test.
It is matching a test where the account was created without `new_buckets_path` and a user is adding it and the test checks that it was added, right?

I think you can check if we have this test, and if we don't you can add a new test.

Reply:
I couldn't find a test that account was created without new_buckets_path and a user is adding it so I added one.
However, for this test we need to get the value of new_buckets_path to its original value for future tests. so I left this code as is.